xmtp/src/client.rs

```diff
@@ -162,7 +165,7 @@ where
 
         let new_installs = contacts
             .iter()
-            .filter(|contact| self_install_id == contact.installation_id())
+            .filter(|contact| self_install_id != contact.installation_id())
```

Comment:
This bug was annoying to catch. Real tests should be added as soon as the infra exists to configure test db states. Currently testing manually via the CLI